docs/integrations/databases/opentelemetry/memcached-opentelemetry.md

```diff
@@ -182,17 +191,83 @@ Following is the query from Errors panel of Memcached app's overview Dashboard:
 | timeslice by 1h 
 | sum(ERROR) as ERROR by _timeslice
 ```
+## Sample metrics query
+**Total Get**
+
+```
+sumo.datasource=memcached deployment.environment=* db.cluster.name=* db.node.name=* metric=memcached.commands command=get | sum
+```
 
-## Viewing Memcached dashboards
+## Create monitors for the Sumo Logic Audit app
```

Comment:
Shouldn't this be the Memcached app?

Reply:
Thanks for pointing that out. Corrected.